stories/topics/proc-azure-monitor-deployment-engine.adoc

```diff
@@ -18,6 +18,11 @@ Approximately 10 minutes into the deployment process, the *Outputs* section of t
 . Login to the deployment using the following credentials:
   * *Username*: _admin_
   * *Password*: Use the _Administrator Password_ that you chose when configuring your deployment.
+. Login to your Red Hat account.
+The account must be associated with a subscription to {PlatformNameShort}.
```

Comment:
Will our customers know what this means or how to ensure that the subscription is associated with the account?

Reply:
Is this really true?  I know it is the goal, but I'm not sure it is a requirement at this stage.  We might check w/ PM.

Reply:
Also, it conflicts with the next line, if they are registering the account fresh, then they won't have a subscription attached yet.

Reply:
Deleted this line. I included it originally because it's mentioned in the modal where you click to log in with your Red Hat account